engine/XcodeUnitTestEngine.php

```diff
@@ -43,16 +44,6 @@ public function shouldEchoTestResults() {
     return false; // i.e. this engine does not output its own results.
   }
 
-  private function shouldGenerateCoverage() {
-    // getEnableCoverage return value meanings:
-    // false: User passed --no-coverage, explicitly disabling coverage.
-    // null:  User did not pass any coverage flags. Coverage should generally be enabled if
-    //        available.
-    // true:  User passed --coverage.
-    // https://secure.phabricator.com/T10561
-    return $this->getEnableCoverage() !== false;
```

Comment:
I like the function actually, and would prefer just add the logic to see whether there is a `coverage` key in the `.arcconfig` file right here. It's just that `loadEnvironment()` would have to be called first to save that info in some type of local variable.